pkgs/top-level/all-packages.nix

```diff
@@ -4689,6 +4689,10 @@ with pkgs;
     inherit (darwin.apple_sdk.frameworks) Security;
   };
 
+  plakar = callPackage ../by-name/pl/plakar/package.nix {
+    buildGoModule = buildGo123Module;
+  };
+
```

Comment:
I think packages in `pkgs/by-name/**/**/package.nix` do not need to be added to this file.

Reply:
where should i add it then? i found other `pkgs/by-name/**/**/package.nix` in that file

Reply:
> Packages found in the name-based structure are automatically included, without needing to be added to all-packages.nix. However if the implicit attribute defaults need to be changed for a package, this [must still be declared in all-packages.nix](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults).

(Source: https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md)

You can use `buildGo123Module` directly in `package.nix` without manually passing it from `all-packages.nix`

Reply:
Having it in `all-packages.nix` is better, we can remove it when the default Go version is bumped.

Reply:
is it okay? should i remove it from all-packages.nix?

Reply:
As [explained in the docs](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md#changing-implicit-attribute-defaults), keeping `buildGoModule` as the interface and overriding it to `buildGo123Module` in `all-packages.nix` is the way to go.

Reply:
fine! Thanks for your help :)